integrations/sklearn/Neptune-Scikit-learn.ipynb

```diff
@@ -0,0 +1,2048 @@
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install scikit-learn==0.23.2 neptune-client==0.4.129 neptune-contrib[monitoring]==0.25.0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation",
+     "library_updates"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install scikit-learn neptune-client neptune-contrib[monitoring] --upgrade"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Scikit-learn regression"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 1: Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import load_boston\n",
+    "from sklearn.ensemble import RandomForestRegressor\n",
+    "from sklearn.model_selection import train_test_split"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 2: Create and fit random forest regressor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "rfr = RandomForestRegressor(n_estimators=70, max_depth=7, min_samples_split=3)\n",
+    "\n",
+    "X, y = load_boston(return_X_y=True)\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20, random_state=28743)\n",
+    "\n",
+    "rfr.fit(X_train, y_train)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 3: Create Neptune experiment and log regressor summary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 71
+    },
+    "colab_type": "code",
+    "id": "GvDSBSrOx-R4",
+    "outputId": "667d1b2f-cdef-44f0-f505-832205dea223",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "from neptunecontrib.monitoring.sklearn import log_regressor_summary\n",
+    "\n",
+    "neptune.init('shared/sklearn-integration',\n",
+    "             api_token='ANONYMOUS')\n",
+    "exp = neptune.create_experiment(name='regression-example',\n",
+    "                                tags=['RandomForestRegressor', 'regression'])\n",
+    "\n",
+    "log_regressor_summary(rfr, X_train, X_test, y_train, y_test, experiment=exp)"
```

Comment:
I think we should add a comment with what was logged and that the test prediction .csv can be huge

Reply:
It will be 1k lines.

Reply:
Added comment what was logged